vignettes/datatable-programming.Rmd

```diff
@@ -373,6 +373,29 @@ print(j)
 DT[, j, env = list(j = j)]
 ```
 
+### Injecting functions using strings in `env`
+
+In `data.table`, you can inject functions into your expressions by passing their names as strings in the `env` parameter. This method allows you to use function names directly as strings, and `data.table` will automatically interpret these strings as the corresponding functions. This approach simplifies the process and makes it easier to work with function names dynamically.
```

Comment:
please add a sentence that says you should use env=list(f="mean") and not env=list(f=mean)